src/stylesheets/core/mixins/_button-unstyled.scss

```
@@ -11,7 +11,9 @@
   text-align: left;
 
   &:hover,
-  &:active {
+  &.usa-button--hover,
+  &:active,
+  &.usa-button--active {
```

Let's add that `disabled` override as well...

```suggestion
  &.usa-button--hover,
  &:active,
  &.usa-button--active,
  &:disabled,
  &.usa-button--disabled {
```

---

> Let's add that `disabled` override as well...

Makes sense to me 👍 

One thing I notice is that we'll likely want to override the default [white text color](https://github.com/uswds/uswds/blob/66b4487f1fb8eb754fb325acbaaaf3a587bbcfa7/src/stylesheets/core/mixins/_button-disabled.scss#L4) of a disabled button, as otherwise the button will be invisible 😅  The outline button is a similar example ([source](https://github.com/uswds/uswds/blob/66b4487f1fb8eb754fb325acbaaaf3a587bbcfa7/src/stylesheets/elements/_buttons.scss#L219)), though it applies the disabled text color using `:disabled` (expected) and a custom [`usa-button--outlined-disabled`](https://github.com/uswds/uswds/blob/66b4487f1fb8eb754fb325acbaaaf3a587bbcfa7/src/stylesheets/elements/_buttons.scss#L216) (not so expected).

Here, I'd propose something like:

```scss
&:disabled,
&.usa-button--disabled {
  color: color("disabled");
}
```